grpc-interface-support/src/proto.py

```diff
@@ -12,10 +12,17 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+from typing import List
+
+from proto_schema_parser import ast
+from proto_schema_parser.parser import Parser
+
 
 class ProtoFileHandle:
     def __init__(self, file_path: str):
         self.file_path = file_path
+        self.service_name = self.get_service_name()
```

Comment:
??? Very confusing to provide a getter and provide a property at the same time.

Upon init we should read the ast and store the parsed data in a private variable. The getters for both imports and service_name should just look up the data in the private ast variable